### PR TITLE
[mesheryctl] Fix panic on version check

### LINKED ISSUE #5293: [mesheryctl] Panic on `version` check: MeshKit Errors string
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Screenshots/Logs
```
$ mesheryctl version
      	VERSION     	GITSHA
Client	v0.6.0-rc-5f	054a83eb2
Server	unavailable 	unavailable

Checking for latest version of mesheryctl...

  v0.6.0-rc-5f is not the latest release. Update to v0.6.0-rc-5g.
panic: interface conversion: error is *errors.errorString, not *errors.Error

goroutine 1 [running]:
github.com/layer5io/meshkit/errors.GetCode({0x30d7ca0, 0xc00061fc90})
	/Users/runner/go/pkg/mod/github.com/layer5io/meshkit@v0.5.10/errors/errors.go:90 +0x90
github.com/layer5io/meshkit/logger.(*Logger).Error(0xc00000e688, {0x30d7ca0, 0xc00061fc90})
	/Users/runner/go/pkg/mod/github.com/layer5io/meshkit@v0.5.10/logger/logger.go:72 +0xbb
github.com/layer5io/meshery/mesheryctl/internal/cli/root.glob..func3(0x43daca0, {0x2da4757, 0x0, 0x0})
	/Users/runner/work/meshery/meshery/mesheryctl/internal/cli/root/version.go:129 +0x4c4
github.com/spf13/cobra.(*Command).execute(0x43daca0, {0x45c87b0, 0x0, 0x0})
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:860 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0x43daa20)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:902
github.com/layer5io/meshery/mesheryctl/internal/cli/root.Execute()
	/Users/runner/work/meshery/meshery/mesheryctl/internal/cli/root/root.go:70 +0x49
main.main()
	/Users/runner/work/meshery/meshery/mesheryctl/cmd/mesheryctl/main.go:22 +0x17
```

#### Environment

- Meshery Server Version: stable-v0.6.0-rc-5g
- Meshery Client Version: stable-v0.6.0-rc-5f

---

#### Contributor Resources
- [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by gr455**
**Description**
Replaced Meshkit logger by logrus for logging non-Meshkit errors

This PR fixes #5293

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/internal/cli/root/version.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
 	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/constants"
 	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	log "github.com/sirupsen/logrus"
```
Comment (piyushsingariya) [inline mesheryctl/internal/cli/root/version.go:28]: No no no no no. Not this @gr455 :rofl: 
We're on a movement of removing directly using logrus in the meshery ecosystem. We have to use utils.Log here in mesheryctl which uses logger from meshkit

Comment (gr455) [reply mesheryctl/internal/cli/root/version.go:28]: Oh well, then we also have to change all these errors to meshkit erros

**Comment by warunicorn19** [comment]
@gr455 let's discuss this PR on the meshery dev call.

File: mesheryctl/internal/cli/root/error.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	ErrAddingContextToConfigCode   = "1053"
+	ErrUnmarshallingConfigFileCode = "1054"
+	ErrGettingRequestContextCode   = "1055"
+	ErrInvalidAPIResponseCode      = "1056"
+	ErrUnmarshallingAPIDataCode    = "1057"
+	ErrConnectingToServerCode      = "1058"
+)
+
+var (
+	ErrCreatingConfigFile = errors.New(ErrCreatingConfigFileCode, errors.Alert, []string{"Unable to create config file"}, []string{"Unable to create config file"}, []string{}, []string{})
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/error.go:18]: @hexxdump, we could use some help on these - filling in the probable cause and suggested remediation.

Comment (leecalcote) [reply mesheryctl/internal/cli/root/error.go:18]: @gr455 will you open an issue to track the need for this information for these mesheryctl error codes? And, we'll move forward with merge here.

**Review by leecalcote — APPROVED** [review]

File: mesheryctl/internal/cli/root/error.go (hunk 2 above)
Comment (leecalcote) [reply mesheryctl/internal/cli/root/error.go:18]: Thank you. 👍

**Pull request merged**